index.php

```diff
@@ -23,39 +29,799 @@
+
+<?php } elseif((($page=="joint1") || ($page=="bjoint1")) && ($lang == "ru"))  { ?>
+
+
+    <nav class="navbar navbar-expand-md navbar-dark fixed-top bg-dark">
+      <!-- Mobile Version -->
+      <div class="container"> <a class="navbar-brand" href="/"><i class="fa fa-fw fa-home">&nbsp;главная</i></a>
+        <ul class="navbar-nav">
+          <li class="nav-item dropdown">
+                <div class="ui simple dropdown item">
+                <button class="navbar-toggler" > <span class="navbar-toggler-icon"></span> </button>
+                <div class="menu" style="right: 0; left: auto;">
+                    <a class="item" href="/aboutj1">О проекте</a>
+                    <a class="item" href="/authorsj1">вкладчики</a>
+                    <a class="item" href="/joint1">Словари</a>
+                    <a class="item" href="/helpj1">помощь</a>
+                    <a class="item" href="/fr"><span class="flag-icon flag-icon-fr"></span></a>
+                    <a class="nav-link">изменить словарьe&nbsp;</a>
+                    <a class="item" href="/about"><?php echo $locale["DAF"]; ?></a>
+                    <a class="item" href="/aboutj1"><?php echo $locale["DINAF"]; ?></a>
+                    <a class="item" href="/aboutj"><?php echo $locale["DINAFN"]; ?></a>
+                    <a class="item" href="/about3"><?php echo $locale["DINAFN1"]; ?></a>
+                  </div>
+                </div>
+          </li>
+        </ul>
+        <!-- Desktop Version -->
+        <div class="collapse navbar-collapse text-center justify-content-end" id="navbar2SupportedContent">
+          <ul class="navbar-nav">
+            <ul class="navbar-nav">
+              <li class="nav-item dropdown">
+                    <div class="ui simple dropdown item">
+                      <a class="nav-link dropdown-toggle" href="about" id="navbarDropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">изменить словарьe&nbsp;</a>
+                      <div class="menu">
+                        <a class="item" href="/about"><?php echo $locale["DAF"]; ?></a>
+                        <a class="item" href="/aboutj1"><?php echo $locale["DINAF"]; ?></a>
+                        <a class="item" href="/aboutj"><?php echo $locale["DINAFN"]; ?></a>
+                        <a class="item" href="/about3"><?php echo $locale["DINAFN1"]; ?></a>
+                      </div>
+                    </div>
+              </li>
+            </ul>
+            <li class="nav-item mx-2"> <a class="nav-link" href="aboutj1">О проекте</a> </li>
+            <li class="nav-item mx-2"> <a class="nav-link" href="authorsj1">вкладчики</a> </li>
+            <li class="nav-item"><a class="nav-link" href="joint1">Словари</a></li>
+            <li class="nav-item"><a class="nav-link" href="helpj1">помощь</a></li>
+            <li class="nav-item mx-2"> <a class="nav-link" href="/fr"><span class="flag-icon flag-icon-fr"> </span></a></li>
```

Comment:
Is it possible to have a config file to generate all those menus instead of copying them in the file? I see the same piece of code 4th times already